web/pageComponents/topicPages/AddToCalendar.tsx

```diff
@@ -88,7 +88,11 @@ const AddToCalendar = ({ eventDate, title, location }: AddToCalendarProps) => {
   if (!fileData) return null
   const atc = intl.formatMessage({ id: 'add_to_calendar_event', defaultMessage: 'Add to Calendar' })
   return (
-    <Button {...(fileData && { href: fileData as string, download: `${title.replace(/ /g, '_')}.ics` })}>
+    <Button
+      aria-label={`${atc} ${title}`}
```

Comment:
can you add a `:` here? So end result will be:

`Add to Calendar: Q3 2022 analyst conference` instead of 
`Add to Calendar Q3 2022 analyst conference`

Reply:
or even better: `Add 'Q3 2022 analyst conference' to Calendar `